### PR TITLE
Document tracking issue for rustdoc `show-type-layout`

### PR DESCRIPTION
**Pull request by tgross35**
Tracking issue: https://github.com/rust-lang/rust/issues/113248

@rustbot label +T-rustdoc +A-docs

### CODE REVIEW

**Comment by tgross35** [comment]
Just since you're currently active on zulip

r? @GuillaumeGomez

File: src/doc/rustdoc/src/unstable-features.md [hunk 1]
```diff
@@ -310,6 +310,8 @@ the source.
 
 ### `--show-type-layout`: add a section to each type's docs describing its memory layout
 
+* Trackin issue: [#113248](https://github.com/rust-lang/rust/issues/113248)
```
Comment (jyn514) [inline src/doc/rustdoc/src/unstable-features.md:313]:
```suggestion
* Tracking issue: [#113248](https://github.com/rust-lang/rust/issues/113248)
```

Comment (tgross35) [reply src/doc/rustdoc/src/unstable-features.md]: Corrected to say `Trackin' issue` instead of `Trackin issue`

(just kidding, should be resolved)

**Pull request merged**